examples/index.html

```diff
@@ -70,9 +71,10 @@
         {% for head in heads %}
           <li>[{{ head.pid_value }}]
             <ul>
-            {% for child in head | all_versions %}
+            {% set head_version = head | to_versioning_api(child=False) %}
```

Comment:
Technically, what you get back from the `to_versioning_api` filter, is a PIDVersioning object, so naming accordingly would make it clearer, eg.:

```{% set pid_versioning = head | to_versioning_api(child=False) %}```